src/css/components/panel.scss

```diff
@@ -68,61 +69,44 @@ h4 + .panel {
 .panel-group {
   @include clearfix;
 
-  margin-bottom: 0.65rem;
-
   & + .panel-group {
     margin-top: $grid-4;
   }
 }
 
 .panel-actions {
-  > * {
-    margin-right: 2rem;
+  margin-top: $grid-4;
 
-    &:last-child {
-      margin-right: 0;
-    }
+  .action {
```

Comment:
We could possibly do this with even margins and negative margin on the container (I like this), but this would just prevent the necessity of the `.panel-actions-right .action` nest. I don't _think_ you can apply the margin globally to action since the actions tend to appear in a wide-ish range of locations. This could be regularized, but it isn't regular yet.